data/importer.js

```diff
@@ -0,0 +1,25 @@
+const fs = require('fs');
+const {promisify} = require('util');
+const csv = require('csvtojson'); 
+
+const readFileAsync = promisify(fs.readFile);
+
+class Importer {
+
+    import(path) {
+        return readFileAsync(path)
```

Comment:
I don't see a reason to wrap the async csvtojson functionality in a promisifying logic (csv() works on promises itself).
I believe csv().fromFile() should be enough.